content/articles/geospatial_standards/index.Rmd

```diff
@@ -0,0 +1,90 @@
+---
+title: "Meet some popular open geospatial standards!"
+description: "A short introduction to the powerful GeoPackage, GeoJSON and GeoTIFF standards"
+author: "Floris Vanderhaeghe"
+date: 2019-11-19
+csl: ../inbo.csl
+bibliography: ../reproducible_research.bib
+categories: ["r", "gis"]
+tags: ["gis", "r", "open science"]
+output: 
+    md_document:
+        preserve_yaml: true
+        variant: gfm
+---
+
+Some inspiration for this post came from the beautiful books of @lovelace_geocomputation_2019, @pebesma_edzer_spatial_2019 and @heijmans_spatial_2019, and from various websites.
+
+## Why use open standards?
+
+- Open file standards ease collaboration, portability and compatibility between users, machines and applications.
+- Their (file) structure is fully documented.
+  - Consequently, scientists and programmers can build new software / packages and make innovations that use these standards, while maintaining interoperability with existing applications.
+  - And, it guarantees that your data will still be readable in a hundred years from now, independently of which IT corporation or software is dominant at that time...
+  
+Luckily, quite a list of open standards is available!
+Below, some powerful and widely-used single-file formats are introduced.
```

Comment:
It has the portability advantage (one dataset = one file). It prevents from pieces getting 'lost' when sharing, republishing etc. and it is the simplest way for users to pick up / copy a dataset.

While it maximizes portability, it is not a requisite of an open standard of course. But I'm sure it will have helped these standards in becoming this popular.